vimrc

```diff
@@ -50,11 +45,8 @@ Bundle 'tpope/vim-rails'
 Bundle 'burnettk/vim-angular'
 Bundle 'othree/html5.vim'
 Bundle 'quentindecock/vim-cucumber-align-pipes'
-Bundle 'vim-scripts/vim-terraform'
-Bundle 'rodjek/vim-puppet'
 
 " misc
-Bundle 'tpope/vim-unimpaired'
```

Comment:
`[<space>` and `]<space>` for adding lines above/below current without going into insert mode and `[n` and `]n` for finding conflict markers are the things i use most